src/typeDeclarations.ts

```diff
@@ -1,6 +1,8 @@
 import { ActionOptions } from 'croods/dist/types/typeDeclarations'
 import { Inputs } from 'react-use-form-state'
 
+export type UserStatus = 'pending' | 'visitor' | 'logged'
```

Comment:
```suggestion
export type UserStatus = 'pending' | 'visitor' | 'loggedIn'
```
loggedIn, right? Is 'logged' used with this meaning?